application/src/main/java/org/togetherjava/tjbot/config/FeatureBlacklist.java

```diff
@@ -0,0 +1,34 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Set;
+
+/**
+ * Blacklist of features, use {@link FeatureBlacklist#isEnabled(T)} to test if a feature is enabled.
+ * 
+ * @param <T> the type of the feature identifier
```

Comment:
what is this generic? i dont think its meaningful to expose it, just set it directly to `Feature`

Reply:
There are two reasons:
- First is because there are Feature implementations, which are classes and so the type is Class, while special blacklist is custom and so can only use String
- Second, just in case, in the future if for example we want enums or something else